src/modules/ModuleStaking/index.vue

```diff
@@ -111,23 +109,34 @@ const poolsPaginated = computed<Pool[] | null>(() => {
 
 const poolsFinal = poolsPaginated
 
-const isLoading = PoolsQuery.loading
+const isLoading = computed(() => poolsFinal.value === null)
 
 function handleStakedUnstaked() {
   quickPoll.value = true
 }
 </script>
 
 <template>
-  <div v-bem>
+  <div
+    :class="$style.moduleStaking"
+    class="flex-1 flex flex-col"
+  >
     <div
-      v-if="!pools?.length && !isLoading"
+      v-if="!poolsFinal?.length && !isLoading"
+      :class="$style.empty"
       class="flex-1 flex items-center justify-center"
     >
-      There are no staking pools at the moment
+      {{
+        !pools?.length
+          ? 'There are no staking pools at the moment'
+          : 'There are no staking pools match the current filter'
+      }}
```

Comment:
Same as in `ModuleFarming/index.vue` - with `<template v-if>` it looks easier (imho).

Reply:
Changed